R/LSTdescriptives.R

```diff
@@ -499,12 +499,7 @@ LSTdescriptives <- function(jaspResults, dataset, options, state = NULL) {
 }
 
 .readDataLSTdesc <- function(jaspResults, options) {
-  variable <- unlist(options[["selectedVariable"]])
-  variable <- variable[variable != ""]
-  dataset <- .readDataSetToEnd(columns.as.numeric = variable)
-  df <- data.frame(x = unlist(dataset))
-  df <- na.omit(df)
-  return(df)
+  return( data.frame(x = na.omit( dataset[[ options[["selectedVariable"]] ]] )))
```

Comment:
This function crashes the analysis when I test things on my machine.
- You're trying to manipulate the `dataset` object, but the function doesn't know about `dataset`.
- Passing `dataset` as a function argument seems to solve things.
   - You'll also need to add a `dataset` argument to `.getDataLSTdesc()`.